components/admin_console/admin_settings.jsx

```diff
@@ -142,10 +142,11 @@ export default class AdminSettings extends React.Component {
         return n;
     };
 
-    parseIntNonZero = (str, defaultValue) => {
+    parseIntNonZero = (str, defaultValue, minimumValue) => {
         const n = parseInt(str, 10);
 
-        if (isNaN(n) || n < 1) {
+        const min = minimumValue || 1;
```

Comment:
Can you set the default value in the function declaration like `parseIntNonZero = (str, defaultValue, minimumValue = 1) => {`?

Other than that one small thing, this looks great!

Reply:
Nice catch! I've fixed it, thanks.